presto-main/src/main/java/io/prestosql/sql/planner/plan/RowNumberNode.java

```diff
@@ -63,6 +63,7 @@ public RowNumberNode(
         checkArgument(!orderSensitive || partitionBy.isEmpty(), "unexpected partitioning in order sensitive node");
```

Comment:
Make commit message more explicit: "Verify arguments to RowNumberNode constructor"